### ISSUE TITLE
npm Debug package override not applying

### ISSUE THREAD

**Issue by ben657** [jspm/registry#1058]
Installing npm:debug@^2.6.0 isn't pulling the override from npm/debug@2.6.0.json into my package.json. Doesn't seem to be anything wrong with the override file, not sure what would be wrong on my end though.

**Comment by aluanhaddad** [comment]
Try
```bash
$ jspm install npm:debug@^2.6.0 --force --override
```

**Comment by ben657** [comment]
Using that force does work fine, odd that it wouldn't work in the first place though right?

**Comment by aluanhaddad** [comment]
Yes, that is odd. It should not be necessary in general but it may have been cached incorrectly (for example by on older version of JSPM). This does happen from time to time and is clearly undesirable. It is not related to this specific package or to the registry I would think.
